### PR TITLE
Changing downloadPath to targetPath and couple of other fields to match V0

### PR DESCRIPTION
**Pull request by fadnavistanmay**
Changing targetPath to downloadPath and couple of other fields to match V0 version od the task

### CODE REVIEW

**Comment by fadnavistanmay** [comment]
@owenhuynMSFT

File: Tasks/DownloadPipelineArtifactV1/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -14,14 +14,14 @@
   "loc.input.label.buildVersionToDownload": "Build version to download",
   "loc.input.label.branchName": "Branch name",
   "loc.input.help.branchName": "Specify to filter on branch/ref name, for example: ```refs/heads/develop```.",
-  "loc.input.label.buildId": "Build",
-  "loc.input.help.buildId": "The build from which to download the artifacts.",
+  "loc.input.label.pipelineId": "Build",
```
[the remark this answers is not in the document]
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV1/Strings/resources.resjson/en-US/resources.resjson]: Changed

**Comment by stephenmichaelf** [comment]
Is this a breaking change that needs major version bump?

**Comment by fadnavistanmay** [comment]
> Is this a breaking change that needs major version bump?

Hi Stephen, we have made sure that the agent is backcompatible , so just a new patch should do.

File: Tasks/DownloadPipelineArtifactV0/task.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 151,
+        "Minor": 139,
```
Comment (stephenmichaelf) [inline Tasks/DownloadPipelineArtifactV0/task.json]: Why is this going down?

File: Tasks/DownloadPipelineArtifactV0/task.loc.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 151,
+    "Minor": 139,
```
Comment (stephenmichaelf) [inline Tasks/DownloadPipelineArtifactV0/task.loc.json]: Why is this going down?

File: Tasks/DownloadPipelineArtifactV1/task.json [hunk 4]
```diff
@@ -9,12 +9,12 @@
     "version": {
         "Major": 1,
         "Minor": 1,
-        "Patch": 2
+        "Patch": 1
```
Comment (stephenmichaelf) [inline Tasks/DownloadPipelineArtifactV1/task.json]: Why is this going down?

File: Tasks/DownloadPipelineArtifactV1/task.loc.json [hunk 5]
```diff
@@ -9,12 +9,12 @@
   "version": {
     "Major": 1,
     "Minor": 1,
-    "Patch": 2
+    "Patch": 1
```
Comment (stephenmichaelf) [inline Tasks/DownloadPipelineArtifactV1/task.loc.json]: Why is this going down?

File: Tasks/PublishPipelineArtifactV0/task.json [hunk 6]
```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 151,
+        "Minor": 139,
```
Comment (stephenmichaelf) [inline Tasks/PublishPipelineArtifactV0/task.json]: Why is this going down?

File: Tasks/PublishPipelineArtifactV0/task.loc.json [hunk 7]
```diff
@@ -9,7 +9,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 151,
+    "Minor": 139,
```
Comment (stephenmichaelf) [inline Tasks/PublishPipelineArtifactV0/task.loc.json]: Why is this going down?

File: Tasks/DownloadPipelineArtifactV0/task.json (hunk 2 above)
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV0/task.json]: This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.

File: Tasks/DownloadPipelineArtifactV0/task.loc.json (hunk 3 above)
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV0/task.loc.json]: This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.

File: Tasks/DownloadPipelineArtifactV1/task.json (hunk 4 above)
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV1/task.json]: This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.

File: Tasks/DownloadPipelineArtifactV1/task.loc.json (hunk 5 above)
Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV1/task.loc.json]: This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.

File: Tasks/PublishPipelineArtifactV0/task.json (hunk 6 above)
Comment (fadnavistanmay) [reply Tasks/PublishPipelineArtifactV0/task.json]: This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.

File: Tasks/PublishPipelineArtifactV0/task.loc.json (hunk 7 above)
Comment (fadnavistanmay) [reply Tasks/PublishPipelineArtifactV0/task.loc.json]: This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.

File: Tasks/DownloadPipelineArtifactV1/task.json (hunk 4 above)
Comment (DavidStaheli) [reply Tasks/DownloadPipelineArtifactV1/task.json]: The previous version was `1.1.0`.  Did you mean to go back to that instead of `1.1.1`?

Comment (fadnavistanmay) [reply Tasks/DownloadPipelineArtifactV1/task.json]: Hi David, no, we have introduced a new patch - hence 1.1.1 - this matches with what we have in the agent plugin.

**Pull request merged**